pom.xml

```diff
@@ -263,6 +263,26 @@
         <artifactId>jsr305</artifactId>
         <version>2.0.3</version>
       </dependency>
+      <dependency>
+        <groupId>com.google.cloud</groupId>
+        <artifactId>google-cloud-pubsub</artifactId>
+        <version>0.5.1</version>
+        <exclusions>
+          <exclusion>
+            <groupId>javax.servlet</groupId>
+            <artifactId>servlet-api</artifactId>
+          </exclusion>
+          <exclusion>
+            <groupId>io.netty</groupId>
+            <artifactId>netty-all</artifactId>
+          </exclusion>
+        </exclusions>
+      </dependency>
+      <dependency>
+        <groupId>io.netty</groupId>
+        <artifactId>netty-all</artifactId>
+        <version>4.1.5.Final</version>
```

Comment:
should add a note in here about why we are using this version, linking to the two other issues, so that someone doesn't accidentally change it in the future without being aware.

I'm not sure that we need to care that the same version of netty-all is used in helios-services (where it really matters) and in the system tests (where it probably doesn't matter what version is used), but if the latter works with this change then that is fine too